### ISSUE TITLE
Highlighting console block with python3-only syntax raises warning

### ISSUE THREAD

**Issue by ngoldbaum** [sphinx-doc/sphinx#5770]
### Problem
- Highlighting a block like this will raise a warning:

```
.. doctest::

   >>> a @ b
```

As far as I can tell there's no way to pass the option that pygments wants to use the `python3` lexer for parsing the python code. See e.g. http://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

#### Procedure to reproduce the problem

Build this sphinx project: https://github.com/ngoldbaum/sphinx-bug-project

#### Error logs / results
```
Warning, treated as error:
/Users/goldbaum/Documents/sphinx-bug/source/index.rst:23:Could not lex literal_block as "pycon". Highlighting skipped.
```

#### Expected results
<Describe what to actually do>

Somehow be able to tell pygments to use the python3 lexer. Or just always use it? It's 2018...

### Reproducible project / your project
- https://github.com/ngoldbaum/sphinx-bug-project

### Environment info
- OS: <Unix/Linux/Mac/Win/other with version> Mac
- Python version: 3.7
- Sphinx version: 1.7.6
- pygments: 2.20

**Comment by ngoldbaum** [comment]
The discussion in https://bitbucket.org/birkenfeld/pygments-main/issues/1194/support-python-35s-matrix-multiplication might obviate the need to think about this.

**Comment by tk0miya** [comment]
I just post #5775. I believe it will resolve your problem.
Thanks,

**Comment by tk0miya** [comment]
Fixed by #5775 .
Thank you for reporting!
